spec/api/comment_thread_spec.rb

```diff
@@ -22,10 +22,10 @@ def thread_result(params)
             t.course_id = "abc"
             t.save!
           end
-          rs = thread_result course_id: "abc", sort_order: "asc"
+          rs = thread_result course_id: "abc"
           rs.length.should == 2
           rs.each_with_index { |res, i|
-            check_thread_result_json(nil, @threads["t#{i+1}"], res)
+            check_thread_result_json(nil, @threads["t#{2-i}"], res)
```

Comment:
No. ;)  I know, I didn't love this, but not sure of the cleanest solution.
Variable `i` is just 0 and 1, so this was "t1", "t2", and now it is "t2", "t1".

I could have something like:
```
descending_index = rs.length - i
check_thread_result_json(nil, @threads["t#{descending_index}"], res)
```
Is that any more clear?  Or would you prefer a comment?  Other?